### DISCUSSION TITLE
Remove delete branch confirmation for closed PRs

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#109263] [Mobile]
GitHub Mobile beta for Android
I loaded a merged PR and tapped the delete branch button:

![Screenshot_20240219-152559.png](https://github.com/community/community/assets/2119212/ba30d6e2-3bfb-4266-834f-16500d14637d)

For a merged branch, I can't think of a good reason to require confirmation.

**Comment by iskae** [comment]
Hi @jsoref 👋 

Thank you for reaching out 🙇 Since delete action is a destructive action, we think that with this confirmation dialog we can avoid miss-clicks on Mobile. Even if the branch has been merged, our users can still compare other branches with a merged branch, therefore the action should still be treated as a destructive action and require a confirmation from the user 😊

**Reply by jsoref** [reply]
When a user deletes a branch, it's replaced by a restore branch button. "Delete branch" is a destructive operation only by the most tenuous definition.

Confirmation dialogs should be limited to things that have serious consequences that can't be easily undone. This operation doesn't count.
